pallets/treasury/src/lib.rs

```diff
@@ -304,23 +305,54 @@ pub mod pallet {
 
             // count actors
             let (count_devs, count_execs) =
-                <Beneficiaries<T>>::iter_keys().fold((0, 0), |mut acc, c| {
+                <Beneficiaries<T>>::iter_keys().fold((0, 0), |mut acc: (i64, i64), c| {
                     match c.1 {
                         BeneficiaryRole::Developer => {
-                            acc.0 += 1;
+                            if let Some(v) = acc.0.checked_add(1) {
+                                acc.0 = v
+                            }
+                            // cannot return Err
```

Comment:
create todos and open corresponding issue

Reply:
Added the TODOs and the issue